kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/Burrow.java

```diff
@@ -116,6 +116,18 @@ private HttpUrl address(String... paths) {
         .collect(Collectors.toList());
   }
 
+  public Map<String, Object> getClusterDetail(String cluster) throws IOException {
```

Comment:
why is this public at all? Testing of burrow should validate that the parse response matches the expected

Reply:
right yeah, should be encapsulated, i'll make it private

Reply:
done